src/hooks/auth0-provider.tsx

```diff
@@ -64,20 +73,23 @@ const Auth0Provider = ({domain, clientId, children}) => {
   }, [client]);
 
   const authorize = useCallback(
-    async (...options) => {
+    async (...options: any[]) => {
```

Comment:
Would it make sense to provide a typed options parameter here instead of resorting to any?